packages/generate-html/test/generate-html.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('GenerateHTML', function () {
       expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/EN10168/v0.4.0/template_hbs.html`, 'utf-8'),
       localTemplatePath: `${__dirname}/../../../fixtures/EN10168/v0.4.0/template.hbs`,
       partialsMap: {
-        inspection: 'https://schemas.s1seven.dev/en10168-schemas/v0.4.0/inspection.hbs',
+        inspection: `${__dirname}/../../../fixtures/EN10168/v0.4.1/inspection.hbs`,
```

Comment:
why did you change the inspection partial path for EN10168 v0.4.0 ?

Reply:
Nice catch, that was only supposed to be for v0.4.1